src/clj/puppetlabs/services/config/puppet_server_config_core.clj

```diff
@@ -92,6 +93,18 @@
                     "conflict with the values to be "
                     "read from puppet: " key-conflicts))))))
 
+(defn check-master-url-prefix
```

Comment:
No big deal, but our pattern for these kinds of functions has generally been to name them like `validate-master-url-prefix!` to signify that they will basically bring the program down, and that they don't return anything but are used for the side effect (in this case, throwing an exception).